dp/cloud/go/services/dp/obsidian/models/conversion.go

```diff
@@ -49,24 +56,42 @@ func CbsdFromBackend(details *protos.CbsdDetails) *Cbsd {
 			MaxPower:         &details.Data.Capabilities.MaxPower,
 			MinPower:         &details.Data.Capabilities.MinPower,
 			NumberOfAntennas: details.Data.Capabilities.NumberOfAntennas,
+			MaxIbwMhz:        details.Data.Capabilities.MaxIbwMhz,
 		},
 		FrequencyPreferences: FrequencyPreferences{
 			BandwidthMhz:   details.Data.Preferences.BandwidthMhz,
 			FrequenciesMhz: makeSliceNotNil(details.Data.Preferences.FrequenciesMhz),
 		},
-		CbsdID:            details.CbsdId,
-		DesiredState:      details.Data.DesiredState,
-		FccID:             details.Data.FccId,
-		Grant:             getGrant(details.Grant),
-		ID:                details.Id,
-		IsActive:          details.IsActive,
-		SerialNumber:      details.Data.SerialNumber,
-		State:             details.State,
-		UserID:            details.Data.UserId,
-		SingleStepEnabled: details.Data.SingleStepEnabled,
-		CbsdCategory:      details.Data.CbsdCategory,
-		InstallationParam: getModelInstallationParam(details.Data.InstallationParam),
+		GrantRedundancy:           details.Data.GrantRedundancy,
+		CarrierAggregationEnabled: details.Data.CarrierAggregationEnabled,
+		CbsdID:                    details.CbsdId,
+		DesiredState:              details.Data.DesiredState,
+		FccID:                     details.Data.FccId,
+		Grant:                     getGrant(details.Grant),
+		ID:                        details.Id,
+		IsActive:                  details.IsActive,
+		SerialNumber:              details.Data.SerialNumber,
+		State:                     details.State,
+		UserID:                    details.Data.UserId,
+		SingleStepEnabled:         details.Data.SingleStepEnabled,
+		CbsdCategory:              details.Data.CbsdCategory,
+		InstallationParam:         getModelInstallationParam(details.Data.InstallationParam),
+	}
+}
+
+func (m *MutableCbsd) ValidateModel(echo.Context) error {
+	if err := m.Validate(strfmt.Default); err != nil {
+		return err
+	}
+	var res []error
+	if !*m.GrantRedundancy && *m.CarrierAggregationEnabled {
+		err := fmt.Errorf("grant_redundancy cannot be set to false when carrier_aggregation_enabled == true")
```

Comment:
since this is supposed to be seen by the end user, I'd rather used "is enabled" or "is true" instead of "== true"